diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2a836c9..3be590e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -58,8 +58,8 @@ ELSE(CURL_FOUND)
   MESSAGE(FATAL_ERROR "Could not find the CURL library and development files.")
 ENDIF(CURL_FOUND)
 
-find_package(unofficial-curlpp CONFIG REQUIRED)
-list(APPEND requiredlibs unofficial::curlpp::curlpp)
+find_package(curlpp CONFIG REQUIRED)
+list(APPEND requiredlibs curlpp::curlpp)
 
 find_package(nlohmann_json CONFIG REQUIRED)
 list(APPEND requiredlibs nlohmann_json::nlohmann_json)